### PR TITLE
i18n(zh-cn): translate `guides/integrations-guide/alpinejs.mdx`

### PR DESCRIPTION
**Pull request by hellolin-oi**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
  translate `guides/integrations-guide/alpinejs.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/alpinejs.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+该软件包由 Astro 核心团队维护。欢迎提交 issue 或 PR！
+
+## 更新日志
+
+请参阅 [CHANGELOG.md](https://github.com/withastro/astro/tree/main/packages/integrations/alpinejs/CHANGELOG.md) 了解此集成的更改历史。
+
+[astro-integration]: /zh-cn/guides/integrations-guide/
+
+[astro-ui-frameworks]: /zh-cn/core-concepts/framework-components/#using-framework-components
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/alpinejs.mdx:109]:
```suggestion
[astro-ui-frameworks]: /zh-cn/core-concepts/framework-components/#使用框架组件
```

**Review by 100gle — COMMENTED** [review]
需要修改一下最后部分链接所对应的中文版本锚点

File: src/content/docs/zh-cn/guides/integrations-guide/alpinejs.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+type: integration
+title: '@astrojs/alpinejs'
+description: 了解如何使用 @astrojs/alpinejs 框架来在你的 Astro 项目中扩展组件支持。
+githubURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/alpinejs/'
+hasREADME: true
+category: renderer
+i18nReady: true
+---
+
+import Video from '~/components/Video.astro';
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/alpinejs.mdx:11]:
```suggestion
```

File: src/content/docs/zh-cn/guides/integrations-guide/alpinejs.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import DontEditWarning from '~/components/DontEditWarning.astro';
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/alpinejs.mdx:12]:
```suggestion
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Review by liruifengv — APPROVED** [review]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**